web/src/features/fireWeather/components/graphs/TempRHGraph.tsx

```diff
@@ -161,13 +161,35 @@ const TempRHGraph = (props: Props) => {
 
   // Update plotly revision to trigger re-drawing of the plot
   const setRevision = useState(0)[1]
+  const [hoverMode, setHoverMode] = useState<
+    'closest' | 'x' | 'y' | 'x unified' | 'y unified' | false
```

Comment:
do we need `y` and `y unified` at all? We don't handle them and I wouldn't expect that they'll be at all useful in our application.

Reply:
Yeah - I'll drop that. I added them all because they're all valid, but since we don't use them...